README.md

```diff
@@ -35,40 +137,89 @@ The raw output looks something like this:
     }
 ```
 
-# Installation
+### Further reading
 
-#### Install from source
+- For more info about analytics codes and what the GA-4 rollout means for OSINT: [https://digitalinvestigations.substack.com/p/what-the-rollout-of-google-analytics]
```

Comment:
```suggestion
- For more info about analytics codes and what the GA-4 rollout means for OSINT: https://digitalinvestigations.substack.com/p/what-the-rollout-of-google-analytics
```
not sure if the brackets were intentional or incomplete markdown, disregard if intentional.